### DISCUSSION TITLE
Change of Password

### DISCUSSION THREAD

**Discussion by Eloquencere** [rustic-rs/rustic#1244] [Q&A] [answered]
Hi, I went through most of the documentation on this link - https://rustic.cli.rs/docs/intro.html  and also most of the help section of the cli but, I can't seem to find the option that lets me change the password of my repo (I remember the original one, I just want to change to something better). It's possible I overlooked something, so please guide me in the right direction

**Comment by aawsome** [comment, marked as answer]
Hi @Eloquencere 
You can add a new key (with a new password) by using `rustic key add`.

So far, there is no `key remove` command. To remove an old key, you have to do this manually in the backend. Just remove the file under `/keys/`.

TL;DR: To change the password, you should run:
- `rustic list keys`
- `rustic add key`
- `rustic list keys`
- manually remove the keyfile which was already present in the first `list keys`

**Comment by Eloquencere** [comment]
Got it! Thanks a lot @aawsome

I see that there is a repo 'init' command but no 'uninit' (if you will). Say I've setup a repo as my backup but, I don't want to use that folder any more for that purpose, how can I un-register it from rustic? is the process just as simple as running - rm repo-bkp/  ?

also, from the documentation I see the format for setting the password file in the config file is something as follows
password-file = "/root/key-rustic"

What should the file format be? And is it just as simple as typing "password" as the first line in the file, also clarify if the quotes are needed
And if I have multiple passwords will they have to be in the same sequence in the file as well? If one password out of the many added passwords isn't present in the password file, does it cause rustic to fail?

**Reply by aawsome** [reply]
There is no `uninit`, but you can just remove the complete repository from the storage backend. You may want to remove a local cache, typically under `~/.cache/rustic/<REPO_ID>`, too.

If you use a password file, just write your password into in cleartext. No quotes or things like this are needed.

If you have multiple keys, you need just one of the passwords. Every key provides a redundant way to access your repository.

**Comment by Eloquencere** [comment]
Great! Thanks a lot for taking the time
 
Loved your answers @aawsome 
You really are AWESOME!
